src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/mapTableToJson.test.ts

```diff
@@ -0,0 +1,476 @@
+import {
+  testTableWithThreeLevelsOfRowAndColHeaders,
+  testTableWithThreeLevelsOfRowAndColHeadersConfig,
+  testTableWithTwoLevelsOfRowAndColHeaders,
+  testTableWithTwoLevelsOfRowAndColHeadersConfig,
+} from '@common/modules/table-tool/utils/__data__/testTableData';
+import mapTableToJson from '@common/modules/table-tool/utils/mapTableToJson';
+
+describe('mapTableToJson', () => {
```

Comment:
Seems like there could be a few extra test cases that would be good to cover:

- a table with one level of row and col headers
- a table with two levels of row headers and one level of col headers
- a table with two levels of col headers and one level of row headers